### PR TITLE
Update weathergov.js - new pull request against develop branch

### PR DESCRIPTION
**Pull request by dgoth**
Fixes https://github.com/MichMich/MagicMirror/issues/3182

Fixed issue with probability of precipitation not showing up on hourly or daily forecast

Hello and thank you for wanting to contribute to the MagicMirror² project

**Please make sure that you have followed these 4 rules before submitting your Pull Request:**

> 1. Base your pull requests against the `develop` branch.
>
> 2. Include these infos in the description:
>
> - Does the pull request solve a **related** issue?
> - If so, can you reference the issue like this `Fixes #<issue_number>`?
> - What does the pull request accomplish? Use a list if needed.
> - If it includes major visual changes please add screenshots.
>
> 3. Please run `npm run lint:prettier` before submitting so that
>    style issues are fixed.
>
> 4. Don't forget to add an entry about your changes to
>    the CHANGELOG.md file.

**Note**: Sometimes the development moves very fast. It is highly
recommended that you update your branch of `develop` before creating a
pull request to send us your changes. This makes everyone's lives
easier (including yours) and helps us out on the development team.

Thanks again and have a nice day!

### CODE REVIEW

**Comment by dgoth** [comment]
Ok I tried creating a new pull request.

Sorry never actually contributed to github before

**Comment by rejas** [comment]
No worries :-) 
Please add a CHANGELOG entry and run the prettier task over the files (and commit again) to get the checks green

File: modules/default/weather/providers/weathergov.js [hunk 1]
```diff
@@ -238,8 +244,7 @@ WeatherProvider.register("weathergov", {
 	 * fetch forecast information for daily forecast.
 	 */
 	fetchForecastDaily(forecasts) {
-		const precipitationProbabilityRegEx = "Chance of precipitation is ([0-9]+?)%";
-
+
```
Comment (rejas) [inline modules/default/weather/providers/weathergov.js:247]: Please run "npm run lint:prettier" and commit the changes.

**Review by rejas — CHANGES_REQUESTED** [review]
Please also add a CHANGELOG entry

**Review by rejas — APPROVED** [review]

**Pull request merged**